src/main/resources/application-test.yml

```
@@ -0,0 +1,29 @@
+  endpoints:
+    web:
+      exposure:
+        include: 'health,info'
+      base-path: '/actuator'
+  endpoint:
+    health:
+      show-details: always
+
+springdoc:
+  api-docs:
+    enabled: false 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)